src/compiler/crystal/tools/doc/method.cr

```diff
@@ -182,9 +182,8 @@ class Crystal::Doc::Method
       name:     name,
       args:     args_to_s,
       doc:      doc,
-      prefix:   prefix,
       abstract: abstract?,
-      kind:     kind,
+      yields:   @def.yields,
```

Comment:
Oh and looks like it's `0` or `1` too, so what do the three values mean? @asterite do you remember?